chemclipse/plugins/org.eclipse.chemclipse.xxd.model/src/org/eclipse/chemclipse/xxd/model/filter/peaks/AsymmetryFilterSettings.java

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Lablicate GmbH.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Alexander Stark - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.chemclipse.xxd.model.filter.peaks;
+
+import org.eclipse.chemclipse.support.settings.DoubleSettingsProperty;
+import org.eclipse.chemclipse.support.settings.EnumSelectionSettingProperty;
+import org.eclipse.chemclipse.xxd.model.support.PeakAsymmetryFilterSelectionCriterion;
+import org.eclipse.chemclipse.xxd.model.support.ValueFilterTreatmentOption;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+
+public class AsymmetryFilterSettings {
+
+	@JsonProperty(value = "As:", defaultValue = "1.02")
+	@JsonPropertyDescription(value = "The peak asymmetry factor of a peak to be filtered accordingly.")
```

Comment:
Add maybe, that this is tailing / leading. Maybe this factor could also be calculated by a strategy, so factor calculation is not fix but could be changed.
e.g. static final Supplier<Double> DefaultfactorSupplier = (p -> .getTailing() / getLeading());
Supplier<Doubl> factorSupplier = DefaultFactorSupplier;
getter/setter

Reply:
I will extend the description.

Can you explain why the factor should be calculated differently or changed later?

In our model we are calculating leading and tailing at 10% peak height. So according to IUPAC the factor is calculated as getTailing() / getLeading(). With each peak that has its own individual leading and tailing values.

If you want to change it later you should maybe consider to implement calculating leading and tailing at 5% peak height in addition to the existing way. Then it would be possible to calculate the factor according to USP, that is (getTailing() + getLeading()) / (2 * getLeading()).

Did you mean this kind of change or did you have something else in mind?